extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -605,3 +605,14 @@ export enum PublishTargetType {
 }
 
 export const CollapseProjectNodesKey = 'collapseProjectNodes';
+
+// httpClient
+export const downloadError = localize('downloadError', "Download error");
+export const downloadProgress = localize('downloadProgress', "Download progress");
+export const downloading = localize('downloading', "Downloading");
+
+// buildHelper
+export const downloadingDacFxDlls = localize('downloadingDacFxDlls', "Downloading Microsoft.Build.Sql nuget to get build dlls");
+export const extractingDacFxDlls = localize('extractingDacFxDlls', "Extracting DacFx build dlls");
+export function errorDownloading(url: string) { return localize('errorDownloading', "Error downloading {0}", url); }
```

Comment:
Can this also include the error message?

Reply:
added